openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Lineage.spec.ts

```diff
@@ -1581,4 +1590,300 @@ test.describe.serial('Test pagination in column level lineage', () => {
       await afterAction();
     }
   });
+
+  test('Verify custom properties tab visibility in lineage sidebar', async ({
+    page,
+  }) => {
+    const { apiContext, afterAction } = await getApiContext(page);
+    const currentTable = new TableClass();
+    const upstreamTable = new TableClass();
+    const downstreamTable = new TableClass();
+
+    try {
+      // Create test entities
+      await Promise.all([
+        currentTable.create(apiContext),
+        upstreamTable.create(apiContext),
+        downstreamTable.create(apiContext),
+      ]);
+
+      await test.step('Create lineage connections', async () => {
+        const currentTableId = currentTable.entityResponseData?.id;
+        const upstreamTableId = upstreamTable.entityResponseData?.id;
+        const downstreamTableId = downstreamTable.entityResponseData?.id;
+
+        if (!currentTableId || !upstreamTableId || !downstreamTableId) {
+          throw new Error('Entity IDs are missing');
+        }
```

Comment:
any reason that we need to throw manual error?
if entity are not created it will auto fail, we dont need manual error.
Please check of all the place.